workspaces/libnpmversion/test/version.js

```diff
@@ -50,13 +54,21 @@ t.test('test out bumping the version in all the ways', async t => {
         },
       }, null, 2),
     },
+    'not-git/package/b': {
+      'npm-shrinkwrap.json': JSON.stringify({
+        ...lock,
+        packages: {
+          '': { ...pkg },
+        },
+      }, null, 2),
+    },
   })
 
   await t.test('git dir', async t => {
     t.afterEach(async () => {
       actionLog.length = 0
     })
-    const path = `${dir}/git`
+    var path = `${dir}/git`
```

Comment:
I don't think this is correct.

Reply:
Fernando made that change already, but I'm curious, since I don't do JS often, so I'm not sure on the styles.  Because it's redeclared later, should we still use let because we're not using strict?  I assumed that coding for strict was preferable.

Reply:
reverted my commit